test/testpulltask.py

```diff
@@ -194,6 +194,8 @@ def run(self, opts, github):
         if not ret:
             proc = subprocess.Popen(cmd)
             ret = testinfra.wait_testing(proc, lambda: self.check_publishing(github))
+            if ret == 124:
```

Comment:
According to the [man page for timeout](http://man7.org/linux/man-pages/man1/timeout.1.html), the return value can be 124 or 128+9, depending on the signal used to end the process.